rskj-core/src/test/java/org/ethereum/util/EthModuleUtils.java

```diff
@@ -0,0 +1,40 @@
+package org.ethereum.util;
```

Comment:
© header missing

Reply:
adding it

Reply:
https://github.com/rsksmart/rskj/pull/1548/commits/dd0463a15e31dbd0e1dd9bdcfbd00c40323bdb63